docs/examples/userguide/language_basics/kwargs_1.pyx

```diff
@@ -0,0 +1,6 @@
+def f(a, b, *args, c, d = 42, e, **kwds):
+    # ...
+    return "hello world"

```

Comment:
I'd rather leave the `...` (which are a perfectly valid function body in Python/Cython) and not have a `return` to keep this really simple and focussed on the signature. No need to assign the return value below then.

Reply:
> which are a perfectly valid function body in Python/Cython
```
>>>mind == 'blown'
True
```